sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -807,6 +807,9 @@ class JDBCSuite extends QueryTest with SharedSparkSession {
       assert(doCompileFilter(LessThan(col0, 5)) === """"col0" < 5""")
       assert(doCompileFilter(LessThan(col0,
         Timestamp.valueOf("1995-11-21 00:00:00.0"))) === """"col0" < '1995-11-21 00:00:00.0'""")
+      assert(doCompileFilter(LessThan(col0,
+        Timestamp.valueOf("2007-12-03 10:15:30").toLocalDateTime))
```

Comment:
It would be better to don't mix classes of different calendar system. Just use the `parse()` method like:
```scala
LocalDateTime.parse("2007-12-03 10:15:30")
```

Reply:
@MaxGekk This parse method don't allow this string format

```
java.time.format.DateTimeParseException: Text '2007-12-03 10:15:30' could not be parsed at index 10
```
but it would be possible in the following way

```
LocalDateTime.parse("2007-12-03 10:15:30", DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"))
```

Reply:
SGTM

Reply:
done!